componenets/button.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Button</title>
+    <link rel="stylesheet" href="/xpressUI.css">
+    <script src="https://kit.fontawesome.com/2ea00fb38c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div id="mySidenav" class="side-nav">
+        <a href="javascript:void(0)" class="closebtn" id="close-sidebar">&times;</a>
```

Comment:
If this is a button then you should use the `<button>` tag instead of the anchor tags